app/scripts/modules/ui/DataView.js

```diff
@@ -228,6 +228,10 @@ DataView.prototype._generateHTMLForKeyValuePair = function (key, currentView) {
         };
     }
 
+    if (typeof vValue === 'string') {
+        vValue = DVHelper.processHTMLString(vValue);
```

Comment:
minor: Do we need the processHTMLString function in order to prepare the HTMLString for the purpose of showing it on the UI? If yes, does it make sense to rename to "formatHTMLString", as 'format' is the more conventional/recognizable name